docker/intel-dgpu-driver.Dockerfile

```diff
@@ -34,6 +34,15 @@ RUN git clone -b ${I915_RELEASE} --single-branch https://github.com/intel-gpu/in
     && cp defconfigs/i915 .config \
```

Comment:
Can we add some links to describe the technical detail in the commit log?

Reply:
Also the related github issue link should be added into the commit log.

Reply:
Done!